package.json

```diff
@@ -162,6 +162,8 @@
 		"freeport": "^1.0.5",
 		"gm": "^1.25.0",
 		"html-entities": "^2.3.2",
+		"i18next": "^23.3.0",
```

Comment:
Wir sollten im Nachhinein nochmal auf das Problem mit der Sprache schauen. Ich finde es sehr ungünstig dass wir uns hiermit die Library ins backend holen müssen. Möglicherweise findet sich irgendein design um das zu vermeiden, wenn auch sicher nicht mehr im scope dieses PRs.

Reply:
Not in scope of this PR